server/http.go

```diff
@@ -21,7 +21,7 @@ import (
 const (
 	scanStatusComplete      = "COMPLETE"
 	scanStatusError         = "ERROR"
-	defaultScanConcurrency  = 5
+	defaultScanConcurrency  = 2
```

Comment:
Are we going to have two simultaneous scans ? How did we arrive at this number ?

Reply:
running more scan simultaneously using more storage while (docker save + untarring  ) especially if image size is big.

and docker takes some time to cleanup the storage if in the mean time container goes out of memory it behaves unpredictable by reducing the total number of simultaneous scans, helping it